### PR TITLE
Fixed different types of bastions not breaking at the same time

### PR DESCRIPTION
**Pull request by xFier**

### CODE REVIEW

**Comment by xFier** [comment]
Removed the unexpectedly highly controversial health messages

**Comment by xFier** [comment]
Opened the PR again

File: src/main/java/isaac/bastion/listeners/BastionDamageListener.java [hunk 1]
```diff
@@ -58,7 +65,52 @@ public void onBlockPlace(BlockPlaceEvent event) {
...
+
+	private void damageRein(BlockPlaceEvent event)
+	{
+		Player p = event.getPlayer();
+		Block b = event.getBlock();
+		Inventory inv = p.getInventory();
+		PlayerState state = PlayerState.get(p);
+		ReinforcementType type = null;
+		Group groupToReinforceTo = null;
+		System.out.println(state.getMode().toString());
```
Comment (Maxopoly) [inline src/main/java/isaac/bastion/listeners/BastionDamageListener.java:81]: Remove this

File: src/main/java/isaac/bastion/listeners/BastionDamageListener.java [hunk 2]
```diff
@@ -58,7 +65,52 @@ public void onBlockPlace(BlockPlaceEvent event) {
 			
 			event.setCancelled(true);
 			event.getPlayer().sendMessage(ChatColor.RED + "Bastion removed block");
+
+			damageRein(event);
+		}
+	}
+
+	private void damageRein(BlockPlaceEvent event)
```
Comment (Maxopoly) [inline src/main/java/isaac/bastion/listeners/BastionDamageListener.java:73]: This entire method is complete bogus, why are you copying Citadel code into Bastion

**Review by Maxopoly — CHANGES_REQUESTED** [review]

**Comment by Maxopoly** [comment]
Not sure if this is still relevant, either way reopen the PR if you ever fix it

**Pull request closed without merging**